### ISSUE TITLE
Option to disable taking the parameter "title" as HTML-Title

### ISSUE THREAD

**Issue by CodingTil** [nuxt/content#1896]
In Nuxt 2, the following markdown file did not change the HTML-Title:
```md
---
title: MyCoolTitle
other_attributes: ...
---
Text
```
If you would have liked to set that title as the HTML-Title, no problem, basically add 1 line of code.

Now, in Nuxt 3, some really narrow-thinking people here thought it would be a great idea to simply overwrite the HTML-Title, **even though** I have the following code:
```js
<script setup>
      useHead({
		title: "TheCoolestTitle",
		titleTemplate: title => title,
	})
</script>
```

While this is a great feature to have for people serving single Markdown Files for single Web Pages in something like a blog, **it makes the library completely unusable for _all other use cases_!**

"Completely unusable?" you might ask? Surely there must be some way to disable this behavior. Surely people have recognized the thousands of projects where people have used this library for Nuxt 2 for applications other than presenting blog posts? _WRONG_

Nowhere in the already very unpolished [Documentation](https://content.nuxtjs.org/) is a workaround mentioned, just a very tiny mention of this behavior! What a shame!

Now, what would happen if I were to change the attribute name from `title` to something like `my_title`? Surely that would work, as no title will be set:
```md
---
my_title: MyCoolTitle
other_attributes: ...
---
Text
```

**Wrong again** This will now set the Markdown-file's **name** as the HTML-Title (slightly modified, underscores removed, and `-` replaced by white space)!

Again, I understand this is a thing, _but **HOW** did no one consider any other use case other than BLOGS_?!

If you seriously cannot name any other use case, here is how I am (trying to) use the Content library:
I have an AboutMe Webpage (not sure if I can link - it's on my git). For this site I store many "CV-Items" in Markdown files. This has the following software-architecture advantages:
 - Separate content from structure
 - Remove complexity
 - Reduce repetitive code by defining how a CV-Item (content) shall be converted to HTML only once
 - Increase maintainability

It is unbelievable that I am not able to set a custom HTML-Title. Strictly speaking, overwriting it should not even be a feature at all in this library (not the purpose), at most optionally overwriting it, **but not by default**.

So please change the default behavior or _at least_ add an boolean flag to disable this behavior!

------

I am in exam season right now, so I don't have the time to try to fix this myself. If this is still not fixed in 1.5 months, I might create a pull request.

I strongly advise in fixing this soon or presenting a workaround (also on the documentation), as I will very probably not be the only one encountering this frustrating issue, that puts projects on hold.

------

The rework of this whole library for Nuxt 3 looks like it just focuses on Blog Articles. The old library was definitely more user friendly (also with respect to modifying the CSS). Also it is bad practice that images should now be placed in the `public` directory, and you know it!
Shame on you for ruining developer experience by purposefully ignoring conventions, practices, and even further!

It is things like these in the NPM / JS world that simply make me want to use Rust's crate [Yew](https://yew.rs/) or Golang's [Vogu](https://www.vugu.org/). Yes, there might not be such a Content library out there, but at least I can still work on my project!

**Comment by CodingTil** [comment]
[`useContentHead()`'s documentation](https://content.nuxtjs.org/api/composables/use-content-head) suggests the following will work, but it doesn't:
```js
<script setup>
	useContentHead({
		head: {
			title: "TheCoolestTitle",
			titleTemplate: title => title,
		},
		title: "TheCoolestTitle",
	})
</script>
```

**Comment by CodingTil** [comment]
> [`useContentHead()`'s documentation](https://content.nuxtjs.org/api/composables/use-content-head) suggests the following will work, but it doesn't:
> 
> ```js
> <script setup>
> 	useContentHead({
> 		head: {
> 			title: "TheCoolestTitle",
> 			titleTemplate: title => title,
> 		},
> 		title: "TheCoolestTitle",
> 	})
> </script>
> ```

According to https://github.com/nuxt/content/blob/main/src/runtime/composables/head.ts I don't see why this doesn't work. It still takes any of the rendered Content files...

Perhaps because this piece of code of mine is in the actual page file, while the Content is being rendered in custom components that use the component `Content`?
